rest-api-spec/src/main/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -52,6 +52,35 @@ setup:
   - is_true: test_index.settings
   - is_true: test_index.mappings
 
+---
+"Test include_type_name":
+  - skip:
+      version: " - 6.6.99"
+      reason: the include_type_name parameter is not backported to pre 6.7 versions yet
```

Comment:
Small comment, maybe this should read 'the include_type_name parameter is not supported before 6.7', since we don't intend backport this further.

Reply:
sure, easy to change